prog/target.go

```diff
@@ -68,6 +68,8 @@ type Target struct {
 	// The default ChoiceTable is used only by tests and utilities, so we initialize it lazily.
 	defaultOnce        sync.Once
 	defaultChoiceTable *ChoiceTable
+
+	Files map[string][]string
```

Comment:
We never call it "files" in other places and "files" looks like a too generic name. When somebody will read this code in future and will see "files" I think they will have no idea what it means and will have no association with anything they see before.
Let's call it Globs or something.